README.adoc

```diff
@@ -4,13 +4,595 @@
 
 Metanorma plugin that allows you to access static data structures like JSON, YAML, XML from a Metanorma document
 
-=== Installation
+== Installation
 
 [source,console]
 ----
 $ gem install metanorma-plugin-datastruct
 ----
 
-== Documentation
+== Usage
 
-See http://metanorma.com[]
+In order to use the macros in Metanorma, add the gem gem 'metanorma-plugin-datastruct' in your Gemfile. Or have the gem installed, and Metanorma can use them automatically via the extension :plugin: datastruct.
+
+EXAMPLE:
+
+[source,ruby]
```

Comment:
We want to remove this block because this is internal Metanorma code. Users will never use this.

Reply:
done, removed